sdk/kiosk/src/constants.ts

```diff
@@ -1,10 +1,90 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-/** The Transer Policy Rules package address on testnet */
-export const TESTNET_RULES_PACKAGE_ADDRESS =
-	'bd8fc1947cf119350184107a3087e2dc27efefa0dd82e25a1f699069fe81a585';
+import {
+	resolveFloorPriceRule,
+	resolveKioskLockRule,
+	resolvePersonalKioskRule,
+	resolveRoyaltyRule,
+} from './tx/rules//resolve';
+import { Network, type ObjectArgument, type RuleResolvingParams } from './types';
 
-/** The transfer policy rules package address on mainnet */
-export const MAINNET_RULES_PACKAGE_ADDRESS =
-	'0x434b5bd8f6a7b05fede0ff46c6e511d71ea326ed38056e3bcd681d2d7c2a7879';
+/**
+ * The Transfer Policy rule.
+ */
+export type TransferPolicyRule = {
+	rule: string;
+	network: Network;
+	packageId: string;
+	resolveRuleFunction: (rule: RuleResolvingParams) => ObjectArgument | void;
+	hasLockingRule?: boolean;
+};
+
+export const royaltyRuleAddress: Record<Network, string> = {
```

Comment:
nit: use all caps for global constants: https://google.github.io/styleguide/tsguide.html#identifiers-constants
```suggestion
export const ROYALTY_RULE_ADDRESS: Record<Network, string> = {
```